app/channel.go

```diff
@@ -1749,6 +1749,18 @@ func (a *App) UpdateChannelLastViewedAt(channelIds []string, userId string) *mod
 	return nil
 }
 
+//MarkChanelAsUnreadFromPost will take a post and set the channel as unread from that one.
```

Comment:
Just some minor nitpicking about spacing
```suggestion
// MarkChanelAsUnreadFromPost will take a post and set the channel as unread from that one.
```

Reply:
I can change that, but I have go-lint enabled which is asking me to remove the space. I've switched from go-lint to golangci-lint so it doesn't show that as a problem

Reply:
That's very strange. The godocs in the standard library have a space at the start of each comment, so I would've assume that go-lint would do the same